### PR TITLE
Wired all API calls for Org Setup

### PR DESCRIPTION
**Pull request by Vivek205**
Image from iOS

### CODE REVIEW

File: packages/marketplace-dapp/package.json [hunk 1]
```diff
@@ -45,7 +45,6 @@
     "redux": "^4.0.1",
     "redux-thunk": "^2.3.0",
     "singularitynet-platform-contracts": "^0.3.4",
-    "snet-sdk-web": "^1.0.0-beta.7",
     "utf8": "^3.0.0",
     "validate.js": "^0.13.1",
     "web3": "1.0.0-beta.52"
```
Comment (ksridharbabuus) [inline packages/marketplace-dapp/package.json]: Let us have a task to upgrade the web3 instance to a stable release as the latest one is 1.2.4.

File: packages/publisher-dapp/src/Pages/OrgSetupStatus/VerificationRejected.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  };
+
+  return (
+    <StatusBanner
+      title="Verification rejected."
+      img="http://placehold.it/302x242"
+      description={`You can continue finishing setting up your company details and publish your company entity to the blockchain.
+                    Then you will be ready to create and publish your new AI services to the AI Marketplace. You can also invite
+                team members to help setup and manage your AI services more efficiently.`}
+      actions={[
```
Comment (ksridharbabuus) [inline packages/publisher-dapp/src/Pages/OrgSetupStatus/VerificationRejected.js]: Actions should be different for the rejected flow. Hope you are going to modify while building the flow.

File: packages/publisher-dapp/src/Pages/OrganizationSetup/PublishToBlockchain/index.js [hunk 3]
```diff
@@ -45,15 +46,19 @@ const PublishToBlockchain = ({ classes, handleFinishLater, history }) => {
     }
   };
 
-  const handlePublish = () => {
+  const handlePublish = async () => {
     setAlert({});
     try {
-      dispatch(organizationActions.submitForApproval(uuid));
+      await dispatch(organizationActions.submitForApproval(organization));
+      await dispatch(organizationActions.publishToBlockchain(uuid));
```
Comment (ksridharbabuus) [inline packages/publisher-dapp/src/Pages/OrganizationSetup/PublishToBlockchain/index.js]: Is this API call to be called after the publishing it to Blockchain rather before? I am assuming it is changing the state from Approval to Blockchain_Submitted which is suppose to be after creating in Blockchain. Please check with API and do it accordingly.

File: packages/marketplace-dapp/package.json (hunk 1 above)
Comment (Vivek205) [reply packages/marketplace-dapp/package.json]: https://github.com/singnet/snet-sdk-js/issues/28

File: packages/publisher-dapp/src/Pages/OrgSetupStatus/VerificationRejected.js (hunk 2 above)
Comment (Vivek205) [reply packages/publisher-dapp/src/Pages/OrgSetupStatus/VerificationRejected.js]: Sure that will be handled in the upcoming PRs

**Pull request merged**